pages/learn/develop/hardware/i2c-and-spi.md

```diff
@@ -348,6 +350,16 @@ After the custom device tree has been validated, it can be included in newer bal
 
 Please note that if the changes for your carrier board expand past kernel device-trees, or require modifications to board configuration files like pin multiplexing configuration files or any other device-trees or files used by firmware, these may not be provided by the existing cloud images. During provisioning the resulting configuration changes are stored in the QSPI or in the hardware defined boot partitions, and thus will be replaced with the default values when updating the Host Operating System. Please [contact us](https://www.balena.io/contact-sales) if you would like to use a Jetson carrier board which may not be fully compatible with its' corresponding devkit, or with any of our cloud images for your Jetson module.
 
+### Configurable fan profiles
+
+Jetson Orin Devices running balenaOS revisions newer than <TODO> and supervisor versions greater than <TODO> offer support for configurable fan profiles. You can switch between the two options provided by Jetpack: "quiet" and "cool", by navigating to the "Device Configuration" page on the balenaCloud dashboard and selecting the desired option. The change is performed at runtime, and does not trigger a device reboot. Preloading the [fan profile configuration](/reference/OS/configuration/#fanprofile) before provisioning your device can also be achieved by editing [config.json](/reference/OS/configuration/#about-configjson)
```

Comment:
Selecting the desired option from how? Will it be a dropdown, will it be a checkbox?
What will the option be called on the Device Configuration page.

Reply:
@cywang117 will this be a drop-down that also allows custom strings, or a string box?

Reply:
@vipulgupta2048 it is a string box, I've extended the description in the last commit